fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -175,14 +180,15 @@ protected void readUnlock() {
     }
 
     /**
-     * Attempts to acquire the write lock. If waiting for the read lock times out or waiting for the lock
-     * an exception is thrown and the query is failed. Timeout is equivalent to the timeout on a search. If a
-     * write lock waits that long then the query is likely going to timeout anyways.
+     * Attempts to acquire the write lock. If waiting for the write lock times out or is interrupted an exception is
+     * thrown and the query is failed. Timeout is equivalent to the timeout on a search.
```

Comment:
`Timeout is equivalent to the timeout on search` isn't actually true. The timeout is `searchTimeout * timeoutMultiplier`.